tests/includes/ssh.sh

```diff
@@ -0,0 +1,33 @@
+# This file contains functions to facilitate SSH connections through a jump host.
+# The commands are useful for testing Juju's SSH proxy using both ssh and scp.
+
+ssh_flags=(
+	-o StrictHostKeyChecking=no
+	-o UserKnownHostsFile=/dev/null
+)
+
+# Note the use of ProxyCommand below instead of the simpler -J flag.
+# This is needed because -J issues an ssh process which doesn't
+# inherit the arguements from the parent process. We want both ssh
+# connections to ignore the host key check to ensure the test runs non-interactively.
+
+make_proxy_command() {
+	local jump_host=$1
+	local ssh_port=17022
+
+	echo "ssh ${ssh_flags[*]} -p $ssh_port $jump_host -W %h:%p"
+}
+
+ssh_wrapper_with_proxy() {
```

Comment:
As I was reading ssh.sh in both, it wasn't clear this was running "ssh" command from the name alone. Just "runSSH" is fine no?

Reply:
>I was reading ssh.sh in both

I'm not following?